web/blog/vast-v2.4.1/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: VAST v2.4.1
+description: Faster Initial Results
+authors: dominiklohmann
+date: 2022-12-19
+tags: [release, feather, performance]
+---
+
+[VAST v2.4.1][github-vast-release] improves the performance of queries when VAST
+is under high load, and significantly reduces the time to first result for
+queries with a low selectivity.
+
+[github-vast-release]: https://github.com/tenzir/vast/releases/tag/v2.4.1
+
+<!--truncate-->
+
+## Reading Feather Files Incrementally
+
+VAST's Feather store naïvely used the [Feather reader][feather-reader] from the
+Apache Arrow C++ library in its initial implementation. However, its API is
+rather limited: It does not support reading incrementally. We've swapped this
+out with a more efficient implementation that does.
+
+[feather-reader]: https://github.com/apache/arrow/blob/apache-arrow-10.0.1/cpp/src/arrow/ipc/feather.h#L57-L108
+
+This is best explained visually:
+
+![Incremental Reads](incremental-reads-light.png#gh-light-mode-only)
+![Incremental Reads](incremental-reads-dark.png#gh-dark-mode-only)
+
+Within the scope of a single store, queries take the same amount of time
+overall, but there exist two distinct advantages of this approach:
```

Comment:
Is it really the same performance? The drawing suggests that I/O and compute are now interleaved, which I would imagine is faster due to pipelining.

Reply:
I made it more explicit that this is about a single query in a single store.